Header header
float32 distance_reward
float32 valid_reward
float32 arrive_reward
float32 crash_reward
float32 laser_reward
float32 linear_punish_reward
float32 angular_punish_reward
float32 step_punish_reward
float32 acc_x_punish_reward
float32 acc_yaw_punish_reward
float32 right_turning_reward
float32 total_reward